core/src/test/java/io/questdb/test/griffin/engine/groupby/SampleByTest.java

```diff
@@ -4867,6 +4867,25 @@ public void testSampleByRewriteWith() throws Exception {
         });
     }
 
+    @Test
+    public void testSampleByRunsSequentiallyWithNonConstantFrom() throws Exception {
```

Comment:
nit: might be a good idea to assert exec plan here. That's because currently the test doesn't check if the exec plan is serial or parallel.